sensor/Networking.py

```diff
@@ -0,0 +1,18 @@
+# TCP client example
+import socket
+
+client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
Imho this file is redundant, but if you want to keep it separated then leave it be :P